x/tokenfactory/keeper/grpc_query.go

```diff
@@ -33,3 +33,17 @@ func (k Keeper) DenomsFromCreator(ctx context.Context, req *types.QueryDenomsFro
 	denoms := k.getDenomsFromCreator(sdkCtx, req.GetCreator())
 	return &types.QueryDenomsFromCreatorResponse{Denoms: denoms}, nil
 }
+
+// Checks whether creator address is in create denom fee whitelist
+func (k Keeper) CreatorInDenomFeeWhitelist(ctx context.Context, req *types.QueryCreatorInDenomFeeWhitelistRequest) (*types.QueryCreatorInDenomFeeWhitelistResponse, error) {
```

Comment:
should we expose this via sei-cosmwasm / sei query wasm handler as well? that way contract could also implement a hard check for inclusion in this whitelist before doing some action (otherwise they would get charged the fee, so they wouldnt be able to gate the ability to create denom by whitelist membership)

Reply:
doesn't have to be in this PR btw

Reply:
Yes will address this in follow-up PR to sei-cosmwasm